src/Utilities.php

```diff
@@ -57,6 +57,10 @@ public static function unescape($value)
      */
     public static function binarySidToString($value)
     {
+        if ($value === '') {
```

Comment:
Shouldn’t we just use `empty()`? 🤔

Reply:
Ok, can you switch it to `empty()` and I’ll merge?

Reply:
Done!

Reply:
Thank you!